### DISCUSSION TITLE
Does autogluon support distrbuted data frame / distributed training ?

### DISCUSSION THREAD

**Discussion by TuanNguyen27** [autogluon/autogluon#2539] [Q&A] [answered]
Perhaps an integration with framework like [Horovod](https://horovod.ai/) ?

**Comment by Innixma** [comment, marked as answer]
Not at present, this is something we plan to investigate in 2023.

**Reply by Omer95** [reply]
Hello @Innixma, you mentioned here https://towardsdatascience.com/autogluon-deep-learning-automl-5cdb4e2388ec , "If you have access to multiple machines, AutoGluon can easily distribute its computation across them in order to return trained models more quickly." Is any kind of distributed/parallel processing of training possible in the current release?
